RELEASE/scripts/autoscend/iotms/mr2020.ash

```diff
@@ -755,3 +755,36 @@ boolean auto_handleRetrocape()
 	}
 	return get_property("retroCapeSuperhero") == tempHero && get_property("retroCapeWashingInstructions") == tag && have_equipped($item[unwrapped knock-off retro superhero cape]);
 }
+
+boolean auto_buyCrimboCommerceMallItem()
+{
+	if (!auto_is_valid($familiar[Ghost of Crimbo Commerce]))
+	{
+		return false;
+	}
+
+	string ghostItemString = get_property("commerceGhostItem");
+	if (ghostItemString == "")
+	{
+		// haven't triggered the greedy ghost message at least once yet.
+		return false;
+	}
+
+	if (get_property("auto_boughtCommerceGhostItem") == ghostItemString)
+	{
+		// already bought the item.
+		return false;
+	}
+
+	auto_log_info(`Commerce Ghost wants us to buy a {ghostItemString} which will give us roughly {my_level()*25} substats in the next combat with it.`);
```

Comment:
what a ridiculous familiar

Reply:
The most ridiculous part is how it's "optimal" in Standard right now for levelling.

I will be very happy when that is no longer the case.